fastlane_core/lib/fastlane_core/crash_reporter/backtrace_sanitizer.rb

```diff
@@ -0,0 +1,49 @@
+module FastlaneCore
+  class BacktraceSanitizer
+
+    def self.sanitize(type: nil, backtrace: nil)
+      if type == :user_error || type == :crash
+        # If the crash is from `UI` we only want to include the stack trace
+        # up to the point where the crash was initiated.
+        # The two stack frames we are dropping are `method_missing` and
+        # the call to `crash!` or `user_error!`.
```

Comment:
Nice catch. Is that always the case? Should we check it here? What happens if we leave the `method_missing` in the stack? Would that make our crash reports less useful, as they're all grouped into one?